main.tf

```
@@ -22,19 +21,25 @@ resource "aws_iam_policy" "app_runner_service_policy" {
   policy = jsonencode({
     "Version" : "2012-10-17",
     "Statement" : [
+      {
+        "Effect" : "Allow",
+        "Action" : [
+          "ecr:GetAuthorizationToken"
+        ],
+        "Resource" : "*"
```

please restrict resource like 

```suggestion
        "Resource" : "${var.apprunner_label_studio_ecr_image_repository_arn}/*"
```

follow this document policy as long as you can do. At least, this permission is bit too wide
https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-started-reduce-permissions.html